dotnet/src/SemanticKernel.UnitTests/Orchestration/ContextVariablesTests.cs

```diff
@@ -239,36 +239,6 @@ public void GetNameThatDoesNotExistReturnsFalse()
         Assert.Null(value);
     }
 
-    [Fact]
```

Comment:
Keep the test. This should work for cloned Contexts as well.

Reply:
The `Update(ContextVariables)` method that this is testing has been removed.

I'll update the test so that it checks a similar scenario, but calls cv.Clone() instead.